hugegraph-test/src/main/java/com/baidu/hugegraph/core/VertexCoreTest.java

```diff
@@ -283,18 +283,18 @@ public void testAddVertexWithInvalidPropertValueOfFloat() {
         schema.propertyKey("float").asFloat().create();
         schema.vertexLabel("number").properties("float").create();
 
-        Assert.assertThrows(IllegalArgumentException.class, () -> {
-            double value = Float.MAX_VALUE * 2.0d;
-            graph.addVertex(T.label, "number", "float", value);
-        });
-        Assert.assertThrows(IllegalArgumentException.class, () -> {
-            double value = -(Float.MAX_VALUE * 2.0d);
-            graph.addVertex(T.label, "number", "float", value);
-        });
-
-        Assert.assertThrows(IllegalArgumentException.class, () -> {
-            graph.addVertex(T.label, "number", "float", Double.MAX_VALUE);
-        });
+        //Assert.assertThrows(IllegalArgumentException.class, () -> {
+        //    double value = Float.MAX_VALUE * 2.0d;
+        //    graph.addVertex(T.label, "number", "float", value);
+        //});
+        //Assert.assertThrows(IllegalArgumentException.class, () -> {
+        //    double value = -(Float.MAX_VALUE * 2.0d);
+        //    graph.addVertex(T.label, "number", "float", value);
+        //});
+        //
+        //Assert.assertThrows(IllegalArgumentException.class, () -> {
+        //    graph.addVertex(T.label, "number", "float", Double.MAX_VALUE);
+        //});
```

Comment:
ditto

Reply:
this method only tests `InvalidProperty`, test & move to `PropertyCoreTest`,